src/styles/embeds/sass/components/cart.css

```diff
@@ -1,18 +1,20 @@
-$cart-item-height: 65px;
-$cart-item-spacing: 20px;
+:root {
+  --cart-item-height: 65px;
+  --cart-item-spacing: 20px;
+}
 
```

Comment:
Talked to @harisaurus about moving all the variables to the variables file since they all complied to `:root` anyway and there is no more scoping.

He is going to do this in a separate PR after this one is out.